src/send-commercial-metrics.spec.ts

```diff
@@ -141,18 +275,27 @@ describe('send commercial metrics code', () => {
 			]);
 		});
 
-		it('should handle connection properties if they exist', () => {
-			const sentMetrics = mockSendMetrics();
-			const eventTimer = EventTimer.get();
+		it('should handle connection properties if they exist', async () => {
+			mockOnConsent(tcfv2AllConsent);
+
+			await initCommercialMetrics({
+				pageViewId: PAGE_VIEW_ID,
+				browserId: BROWSER_ID,
+				isDev: IS_NOT_DEV,
+				adBlockerInUse: ADBLOCK_NOT_IN_USE,
+				sampling: USER_IN_SAMPLING,
+			});
+
 			// Fix the properties of the event timer for the purposes of this test
+			const eventTimer = EventTimer.get();
 			eventTimer.properties = {
 				downlink: 1,
 				effectiveType: '4g',
 			};
-			setVisibility();
+
+			setVisibility('hidden');
 			global.dispatchEvent(new Event('visibilitychange'));
 
-			expect(sentMetrics).toEqual(true);
```

Comment:
Why strip out these assertions that the return value of `initCommercialMetrics` is what we expect it to be? (Sorry I think I missed this earlier)

Reply:
I wanted to keep the tests very focused and this didn't seem super valuable to me, not worth the extra assertion. Whether or not the metrics actually get sent seemed more important

Reply:
For what it's worth we never actually check this value anywhere! 😀

Reply:
Ok that makes sense - let's remove then